library.js

```diff
@@ -153,11 +160,18 @@ iframely.replace = function(raw, options, callback) {
 
 					var icon = getIcon(embed);
 					var image = getImage(embed);
-					if (image) {
+					var scriptSrc = getScriptSrc(embedHtml);
+					// Allow only `iframe.ly/embed.js` script.
+					var isIframelyWidget = scriptSrc && /iframe\.ly\/embed\.js$/.test(scriptSrc);
+					var isSanitized = !scriptSrc || isIframelyWidget;
```

Comment:
Shouldn't the name here be `isNotSanitized` for sanity?

Reply:
`sanitized` means safe for me here.